packages/runtime/container-runtime/src/opLifecycle/outbox.ts

```diff
@@ -139,6 +150,10 @@ export class Outbox {
 	public submit(message: BatchMessage) {
 		this.maybeFlushPartialBatch();
 
+		if (message.type === ContainerMessageType.BlobAttach) {
```

Comment:
Similar question as above - can this decision be done by caller? Can we expose right methods from this class such that caller can accomplish equivalent of submitBlobAttach()?